test/inductor/test_torchinductor.py

```diff
@@ -13626,6 +13626,49 @@ def forward(self, x):
                 FileCheck().check("cpp_fused_add_0").run(code)
             self.assertEqual(refe_out, test_out)
 
+    @requires_gpu()
+    @parametrize("use_cat", [True, False])
+    def test_copy_non_blocking_is_pinned(self, use_cat):
+        def f(a_list):
+            a_cpu_list = []
+            a_to_cpu_event_list = []
+
+            for a in a_list:
+                a_cpu = a.to(device="cpu", non_blocking=True)
+                a_to_cpu_event = torch.cuda.Event()
```

Comment:
```suggestion
                a_to_cpu_event = torch.Event()
```
Sorry for that, this UT will run on XPU as well, so we expect the UT to be generic.